### PR TITLE
Start work on Higher-Level Review form

### PR DESCRIPTION
**Pull request by Mottie**
## Description

This PR contains the basic structure of the new Higher-Level Review form 20-0996.

It was built using the app generator and modified:
- We would like to get this basic structure up and running on staging.va.gov.
- A withdraw from legacy appeal system component was built, but may be updated in the future to use the uiSchema.
- The lighthouse API is not available yet, so placeholder code is being used.
- A `TODO.md` file has been included as a reminder. It will be removed before it is production ready.

## Testing done

Unit tests added for the introduction page, so far.

## Acceptance criteria
- [ ] Basic structure in place for form 20-0996 (Higher-Level Review).
- [ ] This form is not production ready, make sure it will only be available on staging.va.gov.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by smoonpeyes — APPROVED** [review]
Looks great @Mottie! Exciting to see this up so quickly

**Review by ndsprinkle — APPROVED** [review]

**Comment by ndsprinkle** [comment]
Looks awesome :)

File: src/applications/disability-benefits/996/components/IntroductionPage.jsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+import OMBInfo from '@department-of-veterans-affairs/formation-react/OMBInfo';
+import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
+
+import SaveInProgressIntro from '../../../../platform/forms/save-in-progress/SaveInProgressIntro';
+import CallToActionWidget from '../../../../platform/site-wide/cta-widget';
+import { toggleLoginModal } from '../../../../platform/site-wide/user-nav/actions';
+import { focusElement } from '../../../../platform/utilities/ui';
```
Comment (U-DON) [inline src/applications/disability-benefits/996/components/IntroductionPage.jsx]: Feel free to just make all these paths relative to `platform`; we have module aliasing available to us.

File: src/applications/disability-benefits/996/components/IntroductionPage.jsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+  userWithdrewFromLegacySystem = () => {
+    // TODO: replace placeholder; Use API to get/set value; where should the
+    // value set in the this.props.form?
+    this.setState({ isInLegacySystem: false });
+  };
+
+  render() {
+    const { user } = this.props;
+    // console.log(this.props);
```
Comment (U-DON) [inline src/applications/disability-benefits/996/components/IntroductionPage.jsx]: Should probably remove these commented lines once you're done with them.

**Review by U-DON — APPROVED** [review]

File: src/applications/disability-benefits/996/components/IntroductionPage.jsx (hunk 1 above)
Comment (Mottie) [reply src/applications/disability-benefits/996/components/IntroductionPage.jsx]: Done.

File: src/applications/disability-benefits/996/components/IntroductionPage.jsx (hunk 2 above)
Comment (Mottie) [reply src/applications/disability-benefits/996/components/IntroductionPage.jsx]: Removed. I also had a reminder checkbox that I added in the `TODO.md` included in this PR (which will also be removed when ready for production).

**Pull request merged**